doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -323,6 +323,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1
+
+KRKNWK-18495: The Color Control server's RemainingTime attribute change may be reported every 100ms, even though the color temperature value handled by the Color Control server is not modified.
+  This may result in spamming of the Thread network with unnecessary network traffic when controlling :ref:`matter_light_bulb_sample`'s brightness or color.
```

Comment:
```suggestion
  This can result in the Thread network being spammed with unnecessary network traffic when controlling the brightness or color of the :ref:`matter_light_bulb_sample` sample .
```